<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>474</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>484</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>638</width>
    <height>474</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 		rgba(91, 91, 91, 255), stop:1 rgba(118, 118, 118, 255));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Bahnschrift&quot;;
color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>Mostrar Valores</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-160</x>
      <y>110</y>
      <width>231</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: #F0FF42;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;A:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="espacioMatrizA">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: #F0FF42;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;b:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="espacioVectorCoeficientes">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>31</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>31</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: #F0FF42;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;C:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="espacioVectorCostos">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>61</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resorces.qrc"/>
 </resources>
 <connections/>
</ui>
